packages/slate-editor/src/modules/editor/menuOptions.tsx

```diff
@@ -210,6 +210,8 @@ function* generateOptions(
     if (withVideos && withSpecificProviderOptions) {
         yield {
             action: MenuAction.ADD_YOUTUBE,
+            isNew: true,
+            keywords: ['video', 'yt'],
```

Comment:
@alpower I have used these keywords to make the YouTube option show up when users are searching through the [+] menu, letting you know for suggesting any improvements or changes. It would be nice to also do this for any other dedicated option that we're gonna have.

Related comment: https://linear.app/prezly/issue/DEV-11484#comment-72fcc4c9